### PR TITLE
Feature/use public schema

### PR DESCRIPTION
**Pull request by birdNicolas**
major changes:
- use `public` schema
- added `rainfall` function
- table `issues` now using `id` with `serial`-type and `issue_type_id`/`tree_id` using `references` @ff6347 
- added db refactoring (renamings) as preparation for real forecasts
- `forecast_types` has been renamed to `sensor_types` @ff6347 
- added `private` schema

fixes:
- make cron jobs work
- proper use of options in `script_store_radolan_in_db`

### CODE REVIEW

File: infrastructure/database/templates/03_setup_tables.template.sql [hunk 1]
```diff
@@ -90,7 +89,7 @@ CREATE TABLE api.soil (
 );
 
 
-CREATE TABLE "api"."issue_types" (
+CREATE TABLE "public"."issue_types" (
 	"id" int4 NOT NULL,
```
Comment (ff6347) [inline infrastructure/database/templates/03_setup_tables.template.sql:93]: shouldn't that also be `serial`? or 

```sql
id int GENERATED BY DEFAULT AS IDENTITY,
```

File: infrastructure/database/templates/03_setup_tables.template.sql [hunk 2]
```diff
@@ -119,75 +118,103 @@ CREATE TABLE api.weather_stations (
     geometry geometry(POINT,4326)
 );
 
-CREATE TABLE api.weather (
-    STATIONS_ID  BIGINT REFERENCES api.weather_stations (id),
-    MESS_DATUM   timestamp NOT NULL,
+CREATE TABLE public.weather (
+    STATIONS_ID  BIGINT REFERENCES public.weather_stations (id),
```
Comment (ff6347) [inline infrastructure/database/templates/03_setup_tables.template.sql:122]: why uppercase?

File: infrastructure/database/templates/03_setup_tables.template.sql [hunk 3]
```diff
@@ -99,15 +98,15 @@ CREATE TABLE "api"."issue_types" (
...
 	"created_at" timestamptz NOT NULL DEFAULT now(),
-	"tree_id" text NOT NULL,
+	"tree_id" text NOT NULL REFERENCES public.trees (id),
 	PRIMARY KEY ("id")
 );
 
-CREATE TABLE api.weather_stations (
+CREATE TABLE public.weather_stations (
     id   BIGINT PRIMARY KEY,
     von_datum     DATE,
```
Comment (ff6347) [inline infrastructure/database/templates/03_setup_tables.template.sql:111]: why in german here?

File: infrastructure/database/templates/03_setup_tables.template.sql [hunk 4]
```diff
@@ -119,75 +118,103 @@ CREATE TABLE api.weather_stations (
...
+CREATE TABLE public.shading (
+    tree_id TEXT REFERENCES public.trees(id),
     month SMALLINT,
     index FLOAT(53),
     PRIMARY KEY(tree_id, month)
 );
 
-CREATE TABLE api.forecast_types (
+CREATE TABLE public.sensor_types (
 	id SMALLINT PRIMARY KEY,
```
Comment (ff6347) [inline infrastructure/database/templates/03_setup_tables.template.sql:158]: also `serial`

**Review by ff6347 — APPROVED** [review]
Looks good. I added some notes, but they are more stylistic. Might be something for later. If it is up to me you can merge this.

File: infrastructure/database/templates/03_setup_tables.template.sql (hunk 3 above)
Comment (birdNicolas) [reply infrastructure/database/templates/03_setup_tables.template.sql:111]: that was given by the original data source.
we can rename that

**Pull request merged**